packages/client/hmi-client/src/components/tabs/TabContainer.vue

```diff
@@ -22,16 +22,19 @@ const props = defineProps<{
 	activeTabIndex: number;
 }>();
 
-const emit = defineEmits(['closeTab']);
+const emit = defineEmits<{
+	(e: 'tabClosed', index: number): void;
+	(e: 'tabSelected', index: number): void;
```

Comment:
@bencodrington @mwdchang I forgot if we used pascalCase or kebab-case for events?